.github/ISSUE_TEMPLATE/DesignDoc.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Design Doc tracking issue
+about: A standard template to follow when leading a project
+labels: Daily, NewFeature
+---
+
+## Proposal
+
+## Tasks
+
+- [ ] Post **Proposal (Problem/Solution)** in #expensify-open-source
+- [ ] Paste Proposal in the space above with a link to the Slack thread
+- [ ] Email `strategy@expensify.com` and paste in the Proposal
```

Comment:
Why are we asking for strategy@expensify.com here once, and then later, again?

Reply:
Oh sorry, I misread this. It's the same process we have internally, below you're saying use the same email chain you created here. Nevermind.